components/inputs/input-date-time.js

```diff
@@ -0,0 +1,134 @@
+import './input-date.js';
+import './input-fieldset.js';
+import './input-time.js';
+import { convertLocalToUTCDateTime, convertUTCToLocalDateTime } from '@brightspace-ui/intl/lib/dateTime.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { formatDateInISO, formatTimeInISO, parseISODate, parseISODateTime, parseISOTime } from '../../helpers/dateTime.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+export function formatDateTimeInISO(val) {
```

Comment:
I'm torn about if this should throw and if it should be here vs. in the `helpers/dateTime.js` file, since it's only used here.